### DISCUSSION TITLE
How to fix conflicts

### DISCUSSION THREAD

**Discussion by jpitbull917** [SnailyCAD/snaily-cadv4#1489] [Q&A] [answered]
I just moved and got new internet. How do I fix the CAD
![Screenshot 2023-01-28 230851](https://user-images.githubusercontent.com/122257444/215304423-1275cff6-aea1-4f12-a631-5144afc10c0f.png)

**Comment by casperiv0** [comment, marked as answer]
It’s in the documentation: https://cad-docs.caspertheghost.me/docs/installations/updating

![35EA7C7F-19F6-4E65-9370-54F136B2747B](https://user-images.githubusercontent.com/53900565/215307723-b50a5d60-1235-4c36-bf8a-e2c8d7dbd63c.jpeg)
